sql/core/src/main/scala/org/apache/spark/sql/parquet/newParquet.scala

```diff
@@ -123,9 +123,9 @@ class DefaultSource
   }
 }
 
-private[parquet] case class Partition(values: Row, path: String)
+private[sql] case class Partition(values: Row, path: String)
 
-private[parquet] case class PartitionSpec(partitionColumns: StructType, partitions: Seq[Partition])
+private[sql] case class PartitionSpec(partitionColumns: StructType, partitions: Seq[Partition])
```

Comment:
These two are now private to `sql` package because I need them to pass Metastore partitioning information into `ParquetRelation2` while converting Metastore Parquet tables, and that part of code resides in `sql.hive` package.